doc/source/user_guide/index.rst

```diff
@@ -33,6 +33,7 @@ Further information on any specific method can be obtained in the
     reshaping
     text
     missing_data
+    duplicates
```

Comment:
should go near / after indexing?

Reply:
My thinking was it's a "messy data" topic, similar to "missing data".